app/src/controllers/administrate/projects/reducer.test.js

```diff
@@ -0,0 +1,143 @@
+import { ASSIGN_TO_RROJECT_SUCCESS, UNASSIGN_FROM_PROJECT_SUCCESS } from 'controllers/user';
+import { FETCH_SUCCESS } from 'controllers/fetch';
+import { SET_PROJECTS_VIEW_MODE, TABLE_VIEW, GRID_VIEW, NAMESPACE } from './constants';
+import { setViewModeReducer, assignProjectReducer, projectFetchReducer } from './reducer';
+
+const VIEW_MODE_INITIAL_STATE = GRID_VIEW;
+const PROJECTS = [
+  {
+    id: 1,
+    projectName: 'superadmin_personal',
+    usersQuantity: 3,
+    launchesQuantity: 36,
+    launchesPerUser: null,
+    uniqueTickets: null,
+    launchesPerWeek: null,
+    lastRun: 1553585182771,
+    creationDate: 1553072627019,
+    entryType: 'PERSONAL',
+    organization: null,
+  },
+];
+const PROJECTS_AFTER_ASSIGN = [
+  {
+    id: 1,
+    projectName: 'superadmin_personal',
+    usersQuantity: 4,
+    launchesQuantity: 36,
+    launchesPerUser: null,
+    uniqueTickets: null,
+    launchesPerWeek: null,
+    lastRun: 1553585182771,
+    creationDate: 1553072627019,
+    entryType: 'PERSONAL',
+    organization: null,
+  },
+];
+
+const PROJECTS_AFTER_UNASSIGN = [
+  {
+    id: 1,
+    projectName: 'superadmin_personal',
+    usersQuantity: 2,
+    launchesQuantity: 36,
+    launchesPerUser: null,
+    uniqueTickets: null,
+    launchesPerWeek: null,
+    lastRun: 1553585182771,
+    creationDate: 1553072627019,
+    entryType: 'PERSONAL',
+    organization: null,
+  },
+];
+
+describe('projects reducer', () => {
+  describe('setViewModeReducer', () => {
+    test('should return initial state', () => {
+      expect(setViewModeReducer(undefined, {})).toBe(VIEW_MODE_INITIAL_STATE);
+    });
+
+    test('should return old state on unknown action', () => {
+      const oldState = TABLE_VIEW;
+      expect(setViewModeReducer(oldState, { type: 'foo' })).toBe(oldState);
+    });
+
+    test('should handle SET_PROJECTS_VIEW_MODE', () => {
+      const payload = TABLE_VIEW;
+      const newState = setViewModeReducer(VIEW_MODE_INITIAL_STATE, {
+        type: SET_PROJECTS_VIEW_MODE,
+        payload,
+      });
+      expect(newState).toEqual(payload);
+    });
+  });
+
+  describe('projectFetchReducer', () => {
```

Comment:
Just a little tip: `fetchReducer` is already tested, so no need to write duplicating tests for every usage